bin/dkr-pull-test-container

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# tag and push test container to ECR repo
+#
+
+set -e
+
+SCRIPT_DIR=$( dirname "$0" )
+source "$SCRIPT_DIR/dkr-config"
+
+docker pull "018852084843.dkr.ecr.eu-west-1.amazonaws.com/${CONTAINER_NAME}:${CONTAINER_VERSION}"
```

Comment:
Why are you pulling from the registry and then tagging.
This appears to be the opposite to convention.

Reply:
It is set up so that you can build a container locally and use it without using the registry at all.  If you pull from the registry it sets a local tag so the scripts work.

Is there are a better way?

Reply:
Well you can but it's not the usual workflow.